config.schema.yml

```
@@ -379,7 +379,7 @@ properties:
                                     type: "boolean"
                                 realnameFormat:
                                     type: "string"
-                                    enum: ["mxid","reverse-mxid"]
+                                    pattern: "^template:(\\$USERID|\\$LOCALPART|\\$DISPLAY|\\$IRCUSER)|^mxid$|reverse-mxid$"
```

The current pattern only allows four templates.
The value in your tests (`template:$IRCUSER_$DISPLAY_$LOCALPART_$USERID_suffix`) would not be allowed.

```suggestion
                                    pattern: "^template:|^mxid$|reverse-mxid$"
```

---

Good catch, thanks!